### ISSUE TITLE
[Suggestion] Thicker pill

### ISSUE THREAD

**Issue by Jnss98b** [DanGLES3/Oxygen-OS-gesture-bar-for-AOSP#1]
Considering the length of this custom gesture bar, I think it would look better if its thickness were doubled or tripled.

**Comment by DanGLES3** [comment]
if i did that it would not be the Oxygen OS gesture bar, i've got all the values straight from the Oxygen OS systemUI

**Comment by Jnss98b** [comment]
Wait really? I swear OOS' bar looks at least doubled in thickness compared to the stock bar in photos

**Comment by Jnss98b** [comment]
Then could you please make a module with Samsung's pill maybe?

**Comment by DanGLES3** [comment]
> Then could you please make a module with Samsung's pill maybe?

well, for that i would need the systemUI apk from OneUI 2, also is Samsung's pill different from AOSP's? (i honestly can't remember it being different)

**Comment by DanGLES3** [comment]
> Wait really? I swear OOS' bar looks at least doubled in thickness compared to the stock bar in photos

yeah, a friend of mine sent me the SystemUI apk from his Oneplus device (he was the one that asked me to make the module)

**Comment by Jnss98b** [comment]
> > Then could you please make a module with Samsung's pill maybe?
> 
> well, for that i would need the systemUI apk from OneUI 2, also is Samsung's pill different from AOSP's? (i honestly can't remember it being different)

Yeah I remember it looks just a little wider than the stock bar (not as much as OOS') and is probably twice as thick. Unfortunately, I only have access to an S6 so I can't really provide you the apk.

**Comment by Jnss98b** [comment]
The navbar overlay named "Q Center Medium Round" in [this substratum theme](https://tmpfiles.org/download/36975/Flowdor359.apk) looks pretty close to it. It would be great if you could make a module with that bar.

**Comment by Jnss98b** [comment]
This screenshot might give you an idea of its size
![Screenshot_20200120-225549](https://user-images.githubusercontent.com/24306400/72775249-9d3b3080-3bdb-11ea-864a-b5d94057ea5b.jpg)

**Comment by Jnss98b** [comment]
@DanGLES3 I have a question. Could you instruct me on how to modify your module so that I can change the pill dimension to my preference?

**Comment by DanGLES3** [comment]
> @DanGLES3 I have a question. Could you instruct me on how to modify your module so that I can change the pill dimension to my preference?

you can use a tool like apktool or apkeditor to modify my overlay, here are the parameters you can add/modify on the overlay's dimens.xml file 


--
  | <dimen name="navigation_handle_radius">dp</dimen> (radius is how tall the pill is, the rest is either self explanatory or i forgot exactly what they do )
  | <dimen name="navigation_handle_bottom">dp</dimen> 
  | <dimen name="navigation_handle_horizontal_margin">dp</dimen>
  | <dimen name="navigation_home_handle_width">dp</dimen

**Comment by Jnss98b** [comment]
Did you have to sign the apk when you rebuilt it with apktool?
